mlflow/tensorflow/__init__.py

```diff
@@ -85,23 +86,29 @@
 # For tracking if the run was started by autologging.
 _AUTOLOG_RUN_ID = None
 
+# File name to which custom objects cloudpickle is saved - used during save and load
+_CUSTOM_OBJECTS_SAVE_PATH = "custom_objects.cloudpickle"
```

Comment:
Is `.cloudpickle` a common extension for cloudpickle, or is it better to use `.pkl`? I think `.pkl` is more standard.

Reply:
This is existing code of saving keras model. If we change file name, we need add branches in loading method for backwards compatibility. Shall we do this ?

Reply:
Ah, didn't realize. `.cloudpickle` is fine if we're already using it :)